cmd/terminal.go

```diff
@@ -70,9 +70,14 @@ const (
 )
 
 var terminalCmd = &cobra.Command{
-	Use:   "terminal DEVICE_ID",
+	Use:   "terminal [DEVICE_ID]",
```

Comment:
Question:
`Device_ID` is not optional is it?

Reply:
> Improve command description, to explain when DEVICE_ID can be
  optional.
  I guess that explains it :p

Reply:
It is. See [here](https://github.com/mendersoftware/mender-cli/blob/master/cmd/terminal.go#L75). And read also the description :)